en/app.ftl

```diff
@@ -321,6 +321,10 @@ faq-question-acceptable-use-answer-measure-rate-limit = Rate-limiting the number
 #   $attrs (string) - specific attributes added to external links
 faq-question-acceptable-use-answer-b-html = Please review our <a href="{ $url }" { $attrs }>Terms of Service</a> for more information.
 
+faq-question-promotional-email-blocking-question = What is promotional email blocking?
+faq-question-promotional-email-blocking-answer = Relay Premium subscribers can enable promotional email blocking. This feature will forward you important emails, such as receipts, password resets and confirmations, while still blocking marketing messages. There is a slight risk that an important message could still be blocked, so we recommend that you not use this feature for very important places like your bank. If an email is blocked, it cannot be recovered.
```

Comment:
- `Relay Premium` is a brand.
- Remove the `,` before `while still blocking marketing message.`

Reply:
I replaced the brand names in e95ec7b. The comma looked correct to me, so I didn't touch that and will defer to Maxx and Chris on that :)

Reply:
@maxxcrawford